learning-progression.md

```diff
@@ -132,54 +132,97 @@ basic.showIcon(IconNames.Butterfly)
+
+_In javascript, words like `true` and `false` are special keywords reserved by the language. These keywords can only be used to represent a specific boolean value. More on booleans in the next step._
+
+Conversely, if the condition provided within the parentheses is `false`, then the statements in the first block of code will be skipped. Instead, the code in the block following `else` will be run.
+
+```javascript
+if (false) { //This block will be ignored.
     basic.showIcon(IconNames.Heart)
     basic.pause(2000)
     basic.showIcon(IconNames.Butterfly)
-} else {
+} else { //The only condition for this block of code to run, is that the above block did not run.
     basic.showIcon(IconNames.Angry)
     basic.pause(2000)
     basic.showIcon(IconNames.Snake)
 }
 ```
 
-#### 1. Study
+Code written as "`if`-`else`" statements is always executed in an either-or fashion. This means that only one of the two blocks will run. So the program will split and only one of the two possible outcomes will be executed, this is known as a conditional "branch" in programming. 
 
 #### 2. Apply
 
+1. Write a program that shows a heart icon if the condition is true and a butterfly if the condition is false.  
+2. See what happens when you replace `true` or `false` with a mathematical relationship that will be evaluated as either true or false.
+For example, a relationship in which one number is greater than (`>`) or less than (`<`) another.  
+
+   Note that if you are trying to express equality, you will need two equals signs: `==`.
+
 #### 3. Present
 
+In the [Lab Notebook](README.md), include:
+1. A short narrative about the experiment.
+2. Short video of the operation of your code in 2.2.2.
 
+In the repository [programs folder](./programs), include:
+1. File `microbit-program-2-2-2.js` with the code you used in task 2.2.2.
 
 ### Step 6: Data types
 
+#### 1. Study
+
+To start thinking about the most basic type of data we can store, it is helpful to use the analogy of a light switch. When the switch is set to the higher power state, it is on, which can be interpretted to mean true or 1. When the switch is flipped to the lower power state, it is off, false, or 0. In this analogy, we can see that the same piece of data (the light switch being on or off) can be used to represent different kinds of information. 
+
+One of the most rudimentary types of data in javascript is a "Boolean," which has two possible values: `true` and `false`. We can apply this in the example that we used in the previous step to switch which block of code in the "`if`-`else`" statement is run.
+
+A variable is a symbol that is used to represent information. In a computer program, a variable can store information that is intended for use later on. The information stored in the variable can change as the program runs. For example, we may create a boolean with the value of `true`, and change it to `false` if a particular condition is met.
```

Comment:
No. A variable is a named unit of storage for data of a particular type. Also, "information" is not correct here. Data is not equivalent to information. In particular, a variable does not "store information".